scripts/jenkins/ardana/manual/lib.sh

```diff
@@ -111,6 +111,11 @@ function is_physical_deploy {
   [[ $ardana_env == qe* ]] || [[ $ardana_env == pcloud* ]]
 }
 
+function is_crowbar {
+  ardana_env=$(get_from_input scenario_name)
+  [[ $ardana_env == crowbar* ]]
```

Comment:
I would prefer to use `[[ $ardana_env =~ crowbar ]]` instead.

Reply:
reworked into 'cloud_product' func